<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenShot</class>
 <widget class="QMainWindow" name="ScreenShot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icon.qrc">
    <normaloff>:/new/prefix1/Icon/Titleicon.ico</normaloff>:/new/prefix1/Icon/Titleicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="ctr_widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>69</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/prefix1/Icon/top.png);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,0,18,0,3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_time">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_tip">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="BtnBack">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Icon.qrc">
             <normaloff>:/new/prefix1/Icon/back.png</normaloff>:/new/prefix1/Icon/back.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnReload">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Icon.qrc">
             <normaloff>:/new/prefix1/Icon/refresh.png</normaloff>:/new/prefix1/Icon/refresh.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnScnSot">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Icon.qrc">
             <normaloff>:/new/prefix1/Icon/scn.png</normaloff>:/new/prefix1/Icon/scn.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnPreview">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Icon.qrc">
             <normaloff>:/new/prefix1/Icon/preview.png</normaloff>:/new/prefix1/Icon/preview.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnUpload">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../Icon.qrc">
             <normaloff>:/new/prefix1/Icon/upload.png</normaloff>:/new/prefix1/Icon/upload.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="scnsotwidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Icon.qrc"/>
 </resources>
 <connections/>
</ui>
